### PR TITLE
SY-1155 Synnax v0.28 Documentation Update

### PR DESCRIPTION
**Pull request by pjdotson**
Minor changes to the documentation for Synnax v0.28:
- Remove Linux from Console download button
- Add control sequence example page to docs
- Add channel page in console
- Section on snapshotting a schematic

### CODE REVIEW

File: docs/site/src/pages/reference/console/channels.mdx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+title: "Channels"
+heading: "Channels"
+description: "Store data in channels."
+---
+
+import { Divider } from "@synnaxlabs/pluto";
+import { Video } from "@/components/Media";
+import Table from "@/components/Table.astro";
+
+A [channel](/reference/concepts/channels) is a data store that represents ordered values
```
Comment (emilbon99) [inline docs/site/src/pages/reference/console/channels.mdx:12]: I think the language `data store` is a bit confusing. We should stick to a consistent definition that is also defined in the concepts pages

File: docs/site/src/pages/reference/console/channels.mdx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+Index channels are used to store timestamps for data channels. When creating a data
+channel, it must be associated with an index channel. When a data channel records a data
+point, that index channel will then store the timestamp for that data point. The
+difference between the two types of channels is explained more in the [channel concepts
+page](/reference/concepts/channels).
+
+<Divider.Divider direction="x" />
+
+## Aliasing Channels
```
Comment (emilbon99) [inline docs/site/src/pages/reference/console/channels.mdx:76]: This is ok for now, but we should eventually release a more detailed page on aliasing channels.

**Review by emilbon99 — APPROVED** [review]
Approved as soon as you address my comments

File: docs/site/src/pages/reference/console/channels.mdx (hunk 1 above)
Comment (pjdotson) [reply docs/site/src/pages/reference/console/channels.mdx]: I changed it to use the same language as the concepts page. I don't love what it says "logical collection of time-ordered data", but I'm not sure what else to call it. I've described channels as the equivalent of columns in a traditional table database, but I don't think that's the best way to describe it, especially for Console users.

File: client/py/examples/control/abort/README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
-To run this example, you'll need three python shells open. In the first shell, start the
-simulated_daq:
+To run this example, you'll need three Python shells open. In the first shell, start the
+simulated data acquisition computer:
```
Comment (Lham42) [inline client/py/examples/control/abort/README.md:2]: i feel like device or system is better wording than computer

File: docs/site/src/pages/reference/console/channels.mdx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+title: "Channels"
+heading: "Channels"
+description: "Store data in channels."
+---
+
+import { Divider } from "@synnaxlabs/pluto";
+import { Video } from "@/components/Media";
+import Table from "@/components/Table.astro";
+
+A [channel](/reference/concepts/channels) is a logical collection of time-ordered data
```
Comment (Lham42) [inline docs/site/src/pages/reference/console/channels.mdx:12]: what does logical in "logical collection" actually add to the explanation? i wouldn't have it there

**Review by Lham42 — APPROVED** [review]
approved after you address comments

**Pull request merged**